Add /hive-conference-2018 page

morajabi opened the pull request.

Ryman commented:
@emmalou if you want to preview it should be here: http://staging.honeypot.co/pr-633/

Ryman commented:
I also dont see any file referencing Hive?

morajabi commented:
@Ryman Sorry about not committing the files related to Hive. 

Seems like the .gitignore is pretty messed up and that was the reason.

Here it is now: 
http://staging.honeypot.co/pr-633/hive-conference-2018/

Re: why I didn't add the source files, the source files are in the [static-website](https://github.com/honeypotio/static-website) repository which we intended to replace this `staticpages` with when we completed the redesign. 

We can't merge the source files here, and it wouldn't help. I copied built files, as @kdamball preferred with the `/women-in-tech-2018` which is now live here: https://www.honeypot.io/women-in-tech-2018/

Ryman commented:
> @Ryman Sorry about not committing the files related to Hive.
> 
> Seems like the .gitignore is pretty messed up and that was the reason.

No worries, thanks for fixing it :) 



> Re: why I didn't add the source files, the source files are in the static-website repository which we intended to replace this staticpages with when we completed the redesign.
> 
> We can't merge the source files here, and it wouldn't help. I copied built files, as @kdamball preferred with the /women-in-tech-2018 which is now live here: https://www.honeypot.io/women-in-tech-2018/

Thanks for the info 👍 Sorry I don't deal much with this static site so I wasn't entirely aware of what the procedure is.

Regarding the page, here's what I need fixed/confirmed before I can merge:
- The privacy policy link in the cookie banner is the wrong link it should be `/pages/privacy`
- @emmalou are you okay with the `contact us` link being a mail directly to kate instead of making an alias for `events` or `hive`?
- The women in tech banner image seems to disappear when changing language
- @emmalou Should we be linking directly to the tickets popup on eventbrite perhaps? i.e. https://hiveconference2018.eventbrite.ie/#tickets (we are only doing this currently from the link in top right)
- The speaker images are broken, but that may just be an issue with the staging environment but I worry that it's due to uncommitted files?
- Should the `Festsaal Kreuzberg, Berlin, October 18th, 2018 ` mention at the top link to the map part of the page?
- Instagram link is wrong, it should be https://www.instagram.com/honeypot.io/
- (small issue) please add a slash to the linkedin link to save a redirect = faster loading

Looking forward to launching the page!
-

Ryman commented on .gitignore:
```diff
@@ -33,6 +33,7 @@ public/
 !public/nl
 !public/women-in-tech-2018
 !public/hive-conference-2018
+!public/app-bd9747292b5de8a761aa.js
```
can you make this `app-*` instead so that changes to the file are still tracked since that `hash` will likely change on file updates?

morajabi replied:
done

Ryman commented:
also please gitignore  `.DS_Store` files you committed several in your last commit :)

emmalou commented:
Thanks @Ryman 

@morajabi are you okay with the contact us link being a mail directly to kate instead of making an alias for events or hive? **Ok!**
The women in tech banner image seems to disappear when changing language
@morajabi Should we be linking directly to the tickets popup on eventbrite perhaps? i.e. https://hiveconference2018.eventbrite.ie/#tickets (we are only doing this currently from the link in top right) **At the tickets park,  yes**

Should the Festsaal Kreuzberg, Berlin, October 18th, 2018 mention at the top link to the map part of the page? **Don't understand this. Can you screenshot?**

Ryman commented:
> Should the Festsaal Kreuzberg, Berlin, October 18th, 2018 mention at the top link to the map part of the page? Don't understand this. Can you screenshot?

![image](https://user-images.githubusercontent.com/994978/43840960-51de7c06-9b22-11e8-92e9-0705388a1c62.png)

It would be good if this was a link to the map part at the bottom of the page rather than having to have people copy-paste

morajabi commented:
I just fixed the issues and added new requests. For issues of images not appearing, that's most likely a caching problem. Try hard refreshing multiple times and trying in another browser :)
http://staging.honeypot.co/pr-633/hive-conference-2018/
